src/TypeScript/Notifications/INodeExpiredNotificationService.cs

```diff
@@ -0,0 +1,27 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarLint.VisualStudio.TypeScript.Notifications
+{
+    public interface INodeExpiredNotificationService
```

Comment:
Node hasn't "expired" - for whatever reason we haven't found a sufficiently up to date version.
`IUnsupportedNodeVersionNotificationService` is longer but more accurate.

Thinking ahead, in a few months time the JS/TS bubble are going to raise the minimum required version again. This means we might need show another notification to the same user. That suggests we'd need a separate notification id, since we'd still want to show the new notification even if the user has said "don't show me again" for the previous notification.

However, we don't need to decide how to do that now.